targets/psa_switch/pswitch_CLI.py

```diff
@@ -37,41 +50,69 @@ def __init__(self, pre_type, standard_client, mc_client, pswitch_client):
                                         standard_client, mc_client)
         self.pswitch_client = pswitch_client
 
+    @handle_bad_input
     def do_set_queue_depth(self, line):
         "Set depth of one / all egress queue(s): set_queue_depth <nb_pkts> [<egress_port>]"
         args = line.split()
-        depth = int(args[0])
+        depth = self.parse_int(args[0])
         if len(args) > 1:
-            port = int(args[1])
+            port = self.parse_int(args[1])
             self.pswitch_client.set_egress_queue_depth(port, depth)
         else:
             self.pswitch_client.set_all_egress_queue_depths(depth)
 
+    @handle_bad_input
     def do_set_queue_rate(self, line):
         "Set rate of one / all egress queue(s): set_queue_rate <rate_pps> [<egress_port>]"
         args = line.split()
-        rate = int(args[0])
+        rate = self.parse_int(args[0])
         if len(args) > 1:
-            port = int(args[1])
+            port = self.parse_int(args[1])
```

Comment:
`self.parse_int(args[1], "egress_port")`

Reply:
thanks for catching - fixing this in next commit